app/models/assessment_user_datum.rb

```diff
@@ -224,6 +224,10 @@ def self.create_modulo_callbacks(params)
     connection.execute insert_sql
   end
 
+  def global_cumulative_grace_days_used
```

Comment:
Just to clarify, the reason you created this public function is because you can't just use the protected function?

Reply:
Yes!